src/test/java/core/test/StructureTest.java

```diff
@@ -106,7 +106,7 @@ private void checkSuperclass(List<Class> figureClasses) {
   private void checkSuperClassInterfaces(Class clazz) {
     Class[] interfaces = clazz.getInterfaces();
     if (interfaces.length == 0) {
-      Assert.fail("Figures parent class " + clazz.getSuperclass().getSimpleName()
+      Assert.fail("FigureType parent class " + clazz.getSuperclass().getSimpleName()
```

Comment:
Be careful when renaming something project-wide and review what you are committing, you shouldn't make changes here

Reply:
You said to name Color - Colortype; I thought that the figures can also be changed because it is also an enum. I fixed it and I want to finish this task and move to the others. My solution to the problem with the name Figures does not work.

Reply:
I tried with different names and there is an error